src/axiosConfig.js

```diff
@@ -1,7 +1,6 @@
 import axios from 'axios';
 
 const { REACT_APP_BASE_URL } = process.env;
-
```

Comment:
@RachelleMaina, is this change necessary?

Reply:
Not really. I have corrected this.